devfiles/no-custom-image/devfile.yaml

```diff
@@ -0,0 +1,39 @@
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty
+parent:
+  uri: "https://github.com/OpenLiberty/application-stack/releases/download/devfile-0.4.1/devfile.yaml"
```

Comment:
So the parent is to inherit the outer loop and the endpoints?   I think in general we are OK to decouple and not depend on the Open Liberty one.   OTOH, it would be nice to reuse the build/run commands... but I guess we can't because of the different M2 location?

Reply:
Thanks for the review Scott. None of the stack devfile commands can be reused because this sample is also targeting a specific m2 location.
Having said that, I made an update based on our conversation. Please take a look.
Thanks.